CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Other
 - update Cargo.lock dependencies
+- update Cargo.toml.template dependencies
```

Comment:
Please, revert this change. Release-plz workflow will take care of it, just name your PR "chore: update..."